sklearn/_config.py

```diff
@@ -260,6 +274,15 @@ def config_context(
 
         .. versionadded:: 1.3
 
+    engine_attributes : str, default=None
+        Enable conversion of estimator attributes to scikit-learn native
+        types by setting to "sklearn_types". By default attributes are
+        stored using engine native types.
```

Comment:
```suggestion
        stored using engine native types so as to avoid wasteful
        data container conversions and memory transfers between host
        and device between the `fit` and `predict`/`transform` calls
        of an engine-aware estimator.
```